packages/tools/devtools/devtools-test-app/src/FluidObject.ts

```diff
@@ -387,4 +389,19 @@ export class AppData extends DataObject {
 			}),
 		);
 	}
+
+	public getRuntime(): IFluidDataStoreRuntime {
```

Comment:
Is this needed? If the dataobject implementation calls devtools registration, it can just pass its protected property.